addons/web/static/src/core/dropdown/dropdown_item.js

```diff
@@ -10,6 +10,10 @@ const ClosingMode = {
 export class DropdownItem extends Component {
     static template = "web.DropdownItem";
     static props = {
+        tag: {
```

Comment:
this new props must be tested and documented

Reply:
This PR contains a commit from another PR: https://github.com/odoo/odoo/pull/187922
Looks like a mistake. You can post the comment on the right PR. Thanks :)

Reply:
@alexkuhn It's not a mistake, the previous PR fix's an issue in this PR